test-case-reporting/app.ts

```diff
@@ -130,4 +130,9 @@ app.post('/report', jsonParser, async (req, res) => {
   }
 });
 
+app.get('/_cron/compute-stats', async (req, res) => {
+  const {count} = req.query;
+  res.send(`Computing pass/fail % for past ${count} runs`);
```

Comment:
nit: while this is a stub, also log this text so you can verify in the logs that the cron job is running as expected.

Reply:
Done.